packages/framework-integration-tests/src/config/config.ts

```diff
@@ -29,5 +31,10 @@ Booster.configure('production', (config: BoosterConfig): void => {
 
   config.appName = 'my-store-' + appNameSuffix
   config.provider = AWS.Provider()
-  config.assets = ['assets', 'assetFile.txt']
+  config.assets = ['assets']
+  config.tokenVerifier = {
+    issuer: 'booster',
+    // Read the content of the public RS256 cert, used to sign the JWT tokens
+    publicKey: fs.readFileSync(path.join(__dirname, '..', '..', 'assets', 'certs', 'public.key'), 'utf8'),
```

Comment:
This is the reck to perform the tests without hitting any endpoint. Even more if your auth provider doesn't provides  you a jwksUri, and instead provides you the public cert you could use it as well. Both options are available.